docs/docs/guides/how-to-promote-content.md

```diff
@@ -51,7 +51,7 @@ style={{ display: 'block', margin: '0 auto 20px auto' }}
 
 :::info
 
-You must be a `developer` and have access to the chart and space in both the `development project` and the `upstream project`. The space must also be public—promoting content in private or restricted access spaces is not supported yet.
+You must be a `developer` and have access to the chart and space in both the `development project` and the `upstream project`.
```

Comment:
perhaps instead of removing the private warning, we can add a new one saying private access will not be copied into prod